client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java

```diff
@@ -626,6 +636,29 @@ public void write(int index, DateTime v) {
         isSet.set(index, true);
     }
 
+    @Override
+    public void write(int index, Geography v) {
+        SchemaProvider.Field field = schema.field(index);
+        PropertyType typeEnum = PropertyType.findByValue(field.type());
+        if (typeEnum == null) {
+            throw new RuntimeException("Incorrect field type " + field.type());
+        }
+        if (typeEnum == PropertyType.GEOGRAPHY) {
+            if (field.geoShape() != 0 && field.geoShape() != v.getSetField()) {
+                throw new RuntimeException("Incorrect geo shape, expect "
+                        + field.geoShape() + ", got "
+                        + v.getSetField());
+            }
+        } else {
+            throw new RuntimeException("Value: " + v + "'s type is unexpected");
+        }
+        org.locationtech.jts.geom.Geometry jtsGeom = convertGeographyToJTSGeometry(v);
+        byte[] wkb = new org.locationtech.jts.io
+                         .WKBWriter(2, ByteOrderValues.LITTLE_ENDIAN)
```

Comment:
Use byteOrder defined in construct function, which is the native order.

Reply:
What is native order? The default byte order of jts's WKBWriter is `BIG_ENDIAN`, I change it to `little_endian` is because nebula-graph's WKBWriter use the machine's byte order as a default, which is of `little_endian`.

Reply:
native order is the machine's byte order. 

you use the little_endian as the default order, what if user's machine's byte order is big_endian?

Reply:
OK, Thanks for your help